qiskit/_compiler.py

```diff
@@ -279,6 +294,54 @@ def load_unroll_qasm_file(filename, basis_gates='u1,u2,u3,cx,id'):
     return circuit_unrolled
 
 
+def best_subset(backend, n_qubits):
+    """Computes the qubit mapping with the best
+    connectivity.
+
+    Parameters:
+        backend (Qiskit.BaseBackend): A QISKit backend instance.
+        n_qubits (int): Number of subset qubits to consider.
+
+    Returns:
+        ndarray: Array of qubits to use for best
+                connectivity mapping.
+    """
+    device_qubits = backend.configuration['n_qubits']
+    cmap = np.asarray(backend.configuration['coupling_map'])
+
+    if n_qubits > device_qubits:
+        raise ValueError('Number of qubits greater than device.')
+    elif n_qubits <= 0:
+        raise ValueError('Number of qubits <= 0.')
+    if n_qubits == 1:
```

Comment:
I would put this check as the first line of the method. So if it's true we are returning fast.

Reply:
Done.